fido/pronom/soap.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+FIDO: Format Identifier for Digital Objects
+
+Copyright 2010 The Open Preservation Foundation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+PRONOM format signatures SOAP calls.
+"""
+import sys
+import urllib.request
```

Comment:
Hi @sevein I'd thought about that and am happy to patch back. I went this route only in the signature preparation stuff. I thought it might be possible to go with a release that had core `fido` CLI work under 2.7 while the signature prep stuff didn't need to worry about backward compatibility so much. Going back isn't much work.

Reply:
You're right! That makes sense.